tests/MongoDB.Driver.Examples/MongoDB.Driver.Examples.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;netcoreapp2.1;netcoreapp3.0;net452</TargetFrameworks>
     <LangVersion>7.3</LangVersion>
```

Comment:
Lost in the merge, needs an additional line:

`<TargetFrameworks Condition="'$(IsWindows)'!='true'">netcoreapp1.1;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>`